maintainers/maintainer-list.nix

```diff
@@ -8493,6 +8493,12 @@
     githubId = 638763;
     name = "Stuart Moss";
   };
+  stunkymonkey = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add stunkymonkey``.